docs/basics/101-180-FAQ.rst

```diff
@@ -369,7 +369,7 @@ If you do not want to invent a description yourself, you can run
 
       Get the dataset
 
-      A DataLad dataset can be "cloned" by running 'datalad clone <url>'.
+      A DataLad dataset can be "cloned" by running ``datalad clone <url>``.
```

Comment:
I'm unsure whether the backticks are really needed in this format of the template

Reply:
ok, will revert